packages/forms-web-app/src/lib/dashboard-functions.js

```diff
@@ -88,37 +97,54 @@ const overdueDocumentNotToBeDisplayed = (dueDocument) => {
  * @param {string} caseReference
  * @returns {string} returns the seven digit appeal number as a string string
  */
-const extractAppealNumber = (caseReference) => {
-	return caseReference.split('/').pop();
-};
+// Only required if using old mongo instance rather than sql v2 api
```

Comment:
let's delete these commented out functions - we won't be bringing them back!

Reply:
Good call